src/cmd-compress

```diff
@@ -120,7 +120,6 @@ def compress_one_builddir(builddir):
 
     for img_format, img in buildmeta['images'].items():
         if img.get('skip-compression', False):
-            print(f"Skipping {img_format}")
```

Comment:
maybe make these debug instead? I do kind of like the information.

Reply:
or maybe we can collate them all and just print once like we do for `skipped_missing`.

Reply:
Sure, updated. :arrow_down: